bundles/remove.rst

```diff
@@ -4,91 +4,53 @@
-files. Be sure to remove the related configuration from these files. You can
-quickly spot bundle configuration by looking for an ``acme_demo`` (or whatever
-the name of the bundle is, e.g. ``fos_user`` for the FOSUserBundle) string in
-the configuration files.
-
-3. Remove the Bundle from the Filesystem
-----------------------------------------
-
-Now you have removed every reference to the bundle in your application, you
-should remove the bundle from the filesystem. The bundle will be located in
-`src/` for example the ``src/Acme/DemoBundle`` directory. You should remove this
-directory, and any parent directories that are now empty (e.g. ``src/Acme/``).
+.. _unregister-the-bundle-in-the-appkernel:
+.. _remove-bundle-configuration:
+.. _remove-bundle-routing:
+.. _remove-the-bundle-from-the-filesystem:
 
-.. tip::
+1. Uninstall the Bundle
+-----------------------
 
-    If you don't know the location of a bundle, you can use the
-    :method:`Symfony\\Component\\HttpKernel\\Bundle\\BundleInterface::getPath` method
-    to get the path of the bundle::
+Execute this command to remove the Composer package associated with the bundle:
 
-        dump($this->container->get('kernel')->getBundle('AcmeDemoBundle')->getPath());
-        die();
+.. code-block:: terminal
 
-3.1 Remove Bundle Assets
-~~~~~~~~~~~~~~~~~~~~~~~~
+    $ composer remove <package-name>
 
-Remove the assets of the bundle in the public/ directory (e.g.
-``public/bundles/acmedemo`` for the AcmeDemoBundle).
+    # the package name is the same used when you installed it. Example:
+    # composer remove friendsofsymfony/user-bundle
 
-4. Remove Integration in other Bundles
---------------------------------------
+If your application uses :doc:`Symfony Flex </setup/flex>`, this command will
+also unregister the bundle from the application and remove its configuration
+files, environment variables defined by the bundle, etc. Otherwise, you need to
+remove all that manually:
 
-Some bundles rely on other bundles, if you remove one of the two, the other
-will probably not work. Be sure that no other bundles, third party or self-made,
-rely on the bundle you are about to remove.
+* Unregister the bundle in the application removing it from the
+  ``config/bundles.php`` file;
+* Remove any configuration file related to the bundle from the ``config/packages/``
+  directories (the files will be called like the bundle; e.g. ``fos_user.yaml``
+  for FOSUserBundle);
+* Remove any routes file related to the bundle from the ``config/routes/``
+  directories (the file names will also match the bundle name; e.g.
+  ``config/routes/fos_js_routing.yaml`` for FOSJsRoutingBundle).
 
-.. tip::
+2. Remove Bundle Assets
+-----------------------
+
+If the bundle included web assets, remove them from the ``public/bundles/``
+directory (e.g. ``public/bundles/nelmioapidoc/`` for the NelmioApiDocBundle).
+
+3. Remove Integration in other Bundles
```

Comment:
whoa, we shouldn't edit third party bundles to remove their integration? I know you don't mean that, but that title seems to indicate it (and I'm also not very sure what you do mean with this section).

As soon as bundle A depends on B, composer will not allow you to remove B so I don't think there is a problem here?

What we can say is that you have to remove the integration in your application. But that's rather obvious...

Reply:
I didn't change that from the original article. See https://symfony.com/doc/current/bundles/remove.html#remove-integration-in-other-bundles